### PR TITLE
Sort out phpunit.xml location

### PR DESCRIPTION
**Pull request by phil-davis**
Part of issue #52 

In PR #75 we accidentally added a ``phpunit.xml`` in the ``tests/unit`` folder when there was already one in the ``tests`` folder.

This PR:

1) Deletes ``tests/phpunit.xml``
2) Fixes the content of ``tests/unit/phpunit.xml`` so it works for this app
3) Moves ``tests/bootstrap.php`` down into ``tests/unit`` folder
4) Adjusts ``.travis.yml`` so the existing PHP unit tests will still run

### CODE REVIEW

File: tests/unit/bootstrap.php [hunk 1]
```diff
@@ -19,4 +19,4 @@
  *
  */
 
-require_once __DIR__ . '/../../../tests/bootstrap.php';
```
Comment (DeepDiver1975) [inline tests/unit/bootstrap.php]: you can drop this file ....

File: tests/unit/phpunit.xml [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
+		<directory suffix=".php">../../twofactor_totp</directory>
+		<exclude>
+			<directory suffix=".php">../../../twofactor_totp/l10n</directory>
+			<directory suffix=".php">../../../twofactor_totp/tests</directory>
+		</exclude>
 	</filter>
 	<logging>
 		<!-- and this is where your report will be written -->
-		<log type="coverage-clover" target="./tests/output/clover-unit.xml"/>
+		<log type="coverage-clover" target="./clover.xml"/>
```
Comment (DeepDiver1975) [inline tests/unit/phpunit.xml]: we try to put all clover output into tests/output and also try to move all phpunit.xml into the repo root

Comment (phil-davis) [reply tests/unit/phpunit.xml]: OK, I will adjust. I was looking at password_policy for clues.

Comment (DeepDiver1975) [reply tests/unit/phpunit.xml]: It's all mixed .. as usual :man_shrugging:

Comment (phil-davis) [reply tests/unit/phpunit.xml]: PR #78

**Pull request merged**